plugin/resurrect/tab_state.lua

```diff
@@ -98,7 +98,10 @@ function pub.restore_tab(tab, tab_state, opts)
 	wezterm.emit("resurrect.tab_state.restore_tab.finished")
 end
 
-function pub.save_tab_action()
+---@param opt_name? string
+---@param date_fmt? string
+function pub.save_tab_action(opt_name, date_fmt)
+	date_fmt = date_fmt and wezterm.strftime(date_fmt) or ""
```

Comment:
Is this needed, as it calls the `save_state` function with the `date_fmt` which again calls the `wezterm.strftime`?